src/sheetFunctions.js

```diff
@@ -560,3 +560,18 @@ function SAN_ETH_SPENT_OVER_TIME (projectSlug, from, to) {
 function SAN_TOP_HOLDERS_PERCENT_OF_TOTAL_SUPPLY (projectSlug, from, to, numberOfHolders) {
   return handleErrors_(topHoldersPercentOfTotalSupply_)(projectSlug, from, to, numberOfHolders)
 }
+
+/**
+ * Returns the values of a metric of our choice
+ * @param {string} metric Name of the metric.
+ * @param {string} projectSlug Name of the asset at sanbase,
+ * which can be found at the end of the URL (eg. the Santiment URL is
+ * https://app.santiment.net/projects/santiment, so the projectSlug would be santiment).
+ * @param {date} from The starting date to fetch the data. Example: DATE(2018, 9, 20)
+ * @param {date} to The ending date to fetch the data. Example: DATE(2018, 9, 21)
+ * @returns {Array} followers count over time.
```

Comment:
Wrong description

Reply:
It turned out that for top holders percent is also wrong, thanks :D